hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CreateTopLevelDagVisitor.java

```diff
@@ -228,8 +229,8 @@ public Vertex onFullScan(FullScanPhysicalRel rel) {
         dagBuildContext.setRel(rel);
         return getJetSqlConnector(table).fullScanReader(
                 dagBuildContext,
-                rel.filter(),
-                rel.projection(),
+                CalciteNode.filter(rel.filter()),
```

Comment:
All the usages could use static import.

Reply:
https://github.com/hazelcast/hazelcast/pull/23530/commits/8df0f28e672d92accf54d639013e431f15677d57